extension/js/agent/utils/knownTypes.js

```diff
@@ -183,17 +183,29 @@
       });
     }
 
-
+    //Marionette 2.0.0 does not have Radio in Wreqr
     if (Agent.patchedBackbone.Wreqr) {
-      _.extend(knownTypes, {
-        'Backbone.Wreqr.Channel': {
-          type: Agent.patchedBackbone.Wreqr.Channel,
-          name: 'wreqr-channel',
-          className: 'Wreqr.Channel',
-          cid: function(obj) { return ''},
-          toString: toString
-        }
-      });
+      if(Agent.patchedBackbone.Wreqr.Channel) {
+        _.extend(knownTypes, {
+          'Backbone.Wreqr.Channel': {
+            type: Agent.patchedBackbone.Wreqr.Channel,
+            name: 'wreqr-channel',
+            className: 'Wreqr.Channel',
+            cid: function(obj) { return ''},
+            toString: toString
+          }
+        });
+      } else {
+        _.extend(knownTypes, {
+          'Backbone.Wreqr': {
+            type: Agent.patchedBackbone.Wreqr.__proto__.constructor,
```

Comment:
Hmm maybe we don't need this. We detect channels because they're new objects that we want to track, but wreqr is something we don't actually patch.